package.json

```diff
@@ -126,11 +126,10 @@
     "@storybook/addon-styling": "^1.3.7",
     "@storybook/nextjs": "^7.6.3",
     "@storybook/react": "^7.6.3",
-    "@storybook/testing-library": "^0.2.2",
     "@tailwindcss/typography": "^0.5.9",
     "@testing-library/jest-dom": "^6.1.5",
     "@testing-library/react": "^14.1.2",
-    "@types/babel__core": "^7.20.0",
+    "@testing-library/user-event": "^14.5.1",
```

Comment:
We weren't using `@storybook/testing-library` for Storybook, but a component test was using one of it's dependencies, `@testing-library/user-event`. I've installed the package we're actually using for component testing now.

https://github.com/open-sauced/app/blob/2cf4ea6d12021267d186590f53f5ca6585d6d53b/components/organisms/DevCardCarousel/dev-card-carousel.test.tsx#L2